### ISSUE TITLE
creating multiple dependency lock files

### ISSUE THREAD

**Issue by neelasha23** [ploomber/soopervisor#89]
This issue is related to [this issue](https://github.com/ploomber/soopervisor/issues/77).
Given multiple dependency files (requirements.txt or environment.yml) we need to automatically generate the corresponding lock files.

**Comment by edublancas** [comment]
good catch. the lock files are generated when using `ploomber install` - so we should implement it over there. For now, let's keep it here until we merge the code to support multiple `requirements.txt`
